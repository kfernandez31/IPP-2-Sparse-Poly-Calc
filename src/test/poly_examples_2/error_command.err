ERROR 1 STACK UNDERFLOW
ERROR 6 STACK UNDERFLOW
ERROR 8 STACK UNDERFLOW
ERROR 13 STACK UNDERFLOW
ERROR 15 STACK UNDERFLOW
ERROR 20 STACK UNDERFLOW
ERROR 22 STACK UNDERFLOW
ERROR 23 STACK UNDERFLOW
ERROR 24 STACK UNDERFLOW
ERROR 25 STACK UNDERFLOW
ERROR 26 STACK UNDERFLOW
ERROR 28 STACK UNDERFLOW
ERROR 30 STACK UNDERFLOW
ERROR 31 STACK UNDERFLOW
ERROR 32 STACK UNDERFLOW
ERROR 33 STACK UNDERFLOW
ERROR 38 STACK UNDERFLOW
ERROR 40 WRONG COMMAND
ERROR 41 WRONG COMMAND
ERROR 42 WRONG COMMAND
ERROR 43 WRONG COMMAND
ERROR 44 WRONG COMMAND
ERROR 45 WRONG COMMAND
ERROR 46 WRONG COMMAND
ERROR 47 WRONG COMMAND
ERROR 48 DEG BY WRONG VARIABLE
ERROR 49 DEG BY WRONG VARIABLE
ERROR 50 DEG BY WRONG VARIABLE
ERROR 51 DEG BY WRONG VARIABLE
ERROR 52 DEG BY WRONG VARIABLE
ERROR 53 DEG BY WRONG VARIABLE
ERROR 54 DEG BY WRONG VARIABLE
ERROR 55 DEG BY WRONG VARIABLE
ERROR 56 AT WRONG VALUE
ERROR 57 AT WRONG VALUE
ERROR 58 AT WRONG VALUE
ERROR 59 AT WRONG VALUE
ERROR 60 AT WRONG VALUE
ERROR 61 AT WRONG VALUE
ERROR 62 AT WRONG VALUE
ERROR 63 AT WRONG VALUE
ERROR 64 AT WRONG VALUE
ERROR 65 AT WRONG VALUE
ERROR 68 WRONG COMMAND
