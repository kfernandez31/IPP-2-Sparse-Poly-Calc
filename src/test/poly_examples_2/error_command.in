PRINT
(-589,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
(-589,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
ADD
PRINT
ADD
POP
POP
(-589,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
(-589,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
SUB
PRINT
SUB
POP
POP
(-589,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
(-589,0)+(-560,1)+(-607,2)+(583,3)+(-855,4)+(-29,5)+(-106,6)+(634,7)+(832,8)+(701,9)
MUL
PRINT
MUL
POP
POP
IS_COEFF
IS_ZERO
CLONE
NEG
(0,1)
IS_EQ
POP
IS_EQ
DEG
DEG_BY 2
AT 2
(3,2)
AT 2
PRINT
POP
POP
(1,1)
AAA
MUL!
POP 
zero
SUBA
DEG BYS 1
DEG  BY 1
DEG	BY 1
DEG_BY X
DEG_BY  1
DEG_BY 1 
DEG_BY 1a
DEG_BY 18446744073709551616
DEG_BY -5
DEG_BY
DEG_BY +1
AT #45
AT a
AT                     5
AT	1
AT 1 
AT 9223372036854775808
AT -9223372036854775809
AT 
AT 4 38
AT +1

# komentarz
print
