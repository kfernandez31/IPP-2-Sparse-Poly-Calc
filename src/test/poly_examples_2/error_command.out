(-1178,0)+(-1120,1)+(-1214,2)+(1166,3)+(-1710,4)+(-58,5)+(-212,6)+(1268,7)+(1664,8)+(1402,9)
0
(346921,0)+(659680,1)+(1028646,2)+(-6934,3)+(722679,4)+(284000,5)+(1535207,6)+(-1589856,7)+(-864281,8)+(-2601300,9)+(-873823,10)+(-958894,11)+(-630890,12)+(-1381374,13)+(184914,14)+(906364,15)+(1581092,16)+(1166464,17)+(491401,18)
12
